content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,72 @@
+
+## Some of our features
+
+### 🤝 Collaboration
+
+Collaborating with multiple users was [first implemented](https://github.com/excalidraw/excalidraw/pull/879) by [idlewinn](https://github.com/idlewinn) and improved gradually from them on. It's end to end encrypted which means that not only our servers can't read the contents, but if it ever happens to fall into someone else's hands they won't be able to read it either.
+
+### 🔒 Your data is encrypted
+
+The [backend](https://github.com/excalidraw/excalidraw-json) support for storing data was implemented by [lipis](https://github.com/lipis) and the client [support for encrypting](https://github.com/excalidraw/excalidraw/pull/642) the data before submitting was done by [vjeux](https://github.com/vjeux). Read more on how we are doing it in [end to end encryption](/end-to-end-encryption/) article.
+
+### 📚 Library
+
+The library was [first implemented](https://github.com/excalidraw/excalidraw/pull/1787) by [petehunt](https://github.com/petehunt). Later after adding support for saving the library as a file, we eventually introduced a public directory for libraries. Visit [libraries.excalidraw.com](https://libraries.excalidraw.com) for more.
+
+### 📊 Excalicharts
+
+While we aim to keep Excalidraw simple to use, sometimes we hide little easter eggs that you need to find out for yourself (or follow our [Twitter account](https://twitter.com/excalidraw)). For example, you can copy any two column data from Excel, Spreadsheets, or comma separated values (CSV) from a text file and paste them into Excalidraw to get a nice chart out of it. The [first implementation](https://github.com/excalidraw/excalidraw/pull/1723) was done by [petehunt](https://github.com/petehunt) and the [improvements](https://github.com/excalidraw/excalidraw/pull/2495) by [lipis](https://github.com/lipis).
+
+### ✍️ Free Draw mode
+
+The [free hand draw](https://github.com/excalidraw/excalidraw/pull/1570) was added by [kbariotis](https://github.com/kbariotis).
+
+### 🏳 Translations
+
+From early on, users were asking for supporting multiple languages. The [final implementation](https://github.com/excalidraw/excalidraw/pull/638) of it was done by [vjeux](https://github.com/vjeux) and the automatic integration with our [Crowdin](https://crowdin.com/project/excalidraw) project was done by [lipis](https://github.com/lipis).
+
```

Comment:
```suggestion

### 💾 File system integration and file handling

In Excalidraw, we use the [browser-nativefs](https://github.com/GoogleChromeLabs/browser-nativefs) library
to integrate with the file system of the operating system. This allows us to support a true open→edit→save
workflow with proper over-saving and save-as on supporting browsers, and a fallback to downloading files
and file uploads on other browsers. Read more about this feature in [tomayac](https://github.com/tomayac)'s
[article](https://blog.excalidraw.com/browser-nativefs/). We also have
[experimental support](https://web.dev/file-handling/#demo) for file type
association, so that when you double-click an `.excalidraw` file in your file explorer, the Excalidraw PWA opens.
```